### ISSUE TITLE
Install breaks path

### ISSUE THREAD

**Issue by mrmangohands** [enigma9o7/smlinux#16]
I just had to explain to someone that their bashrc shouldn't be completely overwriting their path, I found this after looking at your script for five seconds https://github.com/enigma9o7/smlinux/blob/95556e347b85e9d555883659d96fb4cc18f4e734/smlinux#L1019

You didn't escape it so it gets evaluated before being written

**Comment by mrmangohands** [comment]
If you're lucky this breaks the user's system immediately and they have to deal with it

**Comment by Snivyer1910** [comment]
I can confirm, it happened to me

**Comment by enigma9o7** [comment]
argh I'm an idiot I added that cuz someone complained about having to
reboot.

**Comment by enigma9o7** [comment]
Sorry again.  I just commented out where it writes to bashrc.  I'd just
added tha a couple days ago without testing, seems I will need to figure
out correct syntax later, for now I've just reverted it so wont happen til
its fixed.

**Comment by enigma9o7** [comment]
https://github.com/enigma9o7/smlinux/commit/086d1e27a6cb1958d204ff48527775c5064d2c53

Fixed.  Once again sorry about this, if your path got messed up you just need to delete the last line in ~/.bashrc.  

To explain: smlinux tries to install itself into first folder in your path, if its writable without root.  If it requires root, then it'll create ~/bin and put it there, and tell user about it.   Previously, that's all it did; for debian based distros that'll automatically get added to path on reboot. 

I want smlinux to be easy to use and work first time etc so when someone mentioned their confusion with this, I figured I could easily simplify by force adding it to path in users .bashrc.   So I quickly added in a command to add that, without testing, and forgot the appropriate quotes.   Next day you two guys report the problem and next morning I commented it out, finally had a minute to check it out and resolve today - now it does add to .bashrc but does it correctly.
